docs/index.html

```diff
@@ -0,0 +1,63 @@
+
+<!DOCTYPE html>
+<!--
+  ~ Copyright (c) 2016 Goldman Sachs.
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ and Eclipse Distribution License v. 1.0 which accompany this distribution.
+  ~ The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+  ~ and the Eclipse Distribution License is available at
+  ~ http://www.eclipse.org/org/documents/edl-v10.php.
+  -->
```

Comment:
Pretty much I created this html from scratch, but I noticed I copied small portion of the code (PDF configuration) from reveal.js demo code, so agree it makes sense to added a notice in header. 

Updated the header following the suggestion from this thread: https://developer.jboss.org/thread/158262?tstart=0&_sscc=t